zone/zone_base_data.cpp

```diff
@@ -0,0 +1,35 @@
+#include "zone.h"
+
+BaseDataRepository::BaseData Zone::GetBaseData(uint8 level, uint8 class_id)
+{
+	for (const auto& e : m_base_data) {
+		if (e.level == level && e.class_ == class_id) {
+			return e;
+		}
+	}
+
+	return BaseDataRepository::NewEntity();
+}
+
+void Zone::LoadBaseData()
+{
+	const auto& l = BaseDataRepository::All(content_db);
+
+	if (l.empty()) {
+		return;
+	}
```

Comment:
Can remove this, it is helpful to see if we loaded 0 entries at the end with the log statement